packages/remix-serve/cli.ts

```diff
@@ -36,9 +36,14 @@ let onListen = () => {
 };
 
 let app = createApp(buildPath);
+let server: ReturnType<typeof app.listen>;
 
 if (process.env.HOST) {
-  app.listen(port, process.env.HOST, onListen);
+  server = app.listen(port, process.env.HOST, onListen);
 } else {
-  app.listen(port, onListen);
+  server = app.listen(port, onListen);
+}
```

Comment:
This could be simplified to

```suggestion
let server = process.env.HOST
  ? app.listen(port, process.env.HOST, onListen)
  : app.listen(port, onListen);
```

Reply:
Definitely! I was trying to make the fewest changes possible, and this style difference was previously an option too, so I figured it was not using a ternary on purpose - I'll go ahead and push a change for this :+1: